docs/recipes/babel.md

```diff
@@ -153,6 +153,8 @@ You'll need to install `@babel/polyfill` yourself.
 
 AVA does not currently compile source files. You'll have to load [Babel's `register` module](http://babeljs.io/docs/usage/require/), which will compile source files as needed.
 
+To compile source files on-the-fly, use `@babel/register` else, use `babel-register`.
```

Comment:
This line is unnecessary. However, rather than the line at the end of this section, perhaps this should say something like:

> The below example uses `@babel/register` and assumes you're using Babel 7 to compile your source files. If you're using Babel 6, use `babel-register` instead.

What do you think?